### ISSUE TITLE
Building Bullet for Raspberry Pi

### ISSUE THREAD

**Issue by MeFisto94** [jMonkeyEngine/jmonkeyengine#1183]
Related to #1101 
I have the build working on my fork, however I am having some problems with the CI, which is why I am opening this issue to maybe discuss further progress and/or someone willing to try a few things out.

So the problem essentially manifests in https://bugs.launchpad.net/ubuntu/+source/gcc-defaults/+bug/1300211 -> We can EITHER Compile Linux 32bit OR ARM, but not both. Oddly enough it works on my machine and in a docker container just fine, however I don't know if the produced binaries are valid then.

So, clean solutions would be:
1. Using multiple docker containers and building the natives from within travis
2. Splitting of the ARM builds into a separate artifact, as that is what is preventing me from using yet another travis instance/matrix entry.

Workarounds would be:
3. Fixing Travis, so that everything builds on it (locally and in a docker container it works, so the task here would be to spot which dependency might have been removed from travis to save space and add it, though it will be hard. Hint: `dpkg -l > file.txt` lists all installed packages into a file which would then have to be compared with the docker container (or use `dpkg -l` directly to have it print out everything).

4. Using the docker container which executes the following steps:
```
apt update && apt install git openjdk-11-jdk
git clone https://github.com/MeFisto94/jmonkeyengine && cd jmonkeyengine && git checkout feature/bullet-arm
apt install gcc-multilib g++-multilib
apt install gcc-7-arm-linux-gnueabihf gcc-7-arm-linux-gnueabi gcc-8-aarch64-linux-gnu g++-7-arm-linux-gnueabihf g++-7-arm-linux-gnueabi g++-8-aarch64-linux-gnu
./gradlew -PbuildNativeProjects=true jme3-bullet-native:assemble
```
This works as well. Note that the trick here is installing multilib  first and then the cross-compilers which uninstall multilib again but somehow the system keeps what is required to build multilib.

Why are 3 and 4 unclean? Simple: When I check my include path, I see that some 64bit headers are pulled in, which is what causes the build to work. Usually it's multilibs job to somehow manage these problems. It could be that the resulting library is 32bit but crashes because some contracts are broken.

TLDR: ATM it works, but prevents 32 bit Linux from building, so the easiest way would be to separate them

**Comment by stephengold** [comment]
I'm fine with putting the Linux ARM native libraries into a new artifact.

**Comment by MeFisto94** [comment]
Great. While it annoys me that we cannot build them all together, this is by far the most simple solution. I just have to see how we can do that without copying the code from jme3-bullet-native.
I might need someone with gradle-fu for that.

**Comment by riccardobl** [comment]
Done in https://github.com/jMonkeyEngine/jmonkeyengine/pull/1190
